src/commands/InitCommand.js

```diff
@@ -33,19 +33,27 @@ export default class InitCommand extends Command {
   ensurePackageJSON() {
     let {packageJsonLocation, packageJson} = this.repository;
 
-    if (!packageJson) packageJson = {};
-    // if (!packageJson.private) packageJson.private = true;
-    if (!packageJson.devDependencies) packageJson.devDependencies = {};
-
-    objectAssignSorted(packageJson.devDependencies, {
-      lerna: this.lernaVersion
-    });
-
     if (!packageJson) {
```

Comment:
this logic down here was flawed, as we update in case of `!packageJson` a few lines further up.